src/application/service/utils.ts

```diff
@@ -17,9 +17,9 @@ export const incompletePartnerErrorList = (
   let errorList = {};
 
   partners
-    .filter((gp) => gp?.data?.completed === false)
-    .forEach((gp) => {
-      const name = gp.data?.forename ? `${gp.data.forename} ${gp.data.surname}` : gp.data?.legal_entity_name ?? "";
+    .filter((p) => p?.data?.completed === false)
```

Comment:
changed these to p from gp as they can also be limited partners, so using p for partner